postgres/README.md

```diff
@@ -0,0 +1,226 @@
+## PostgreSQL Example Usage
```

Comment:
Same comment as mysql/README.md: I think we should have a top-level header and a bit of explanatory text. How about something like:

# HarbourBridge: Turnkey PostgreSQL-to-Spanner Evaluation

HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation, using data from an existing PostgreSQL or MySQL database. This README provides details of the tool's PostgreSQL capabilities. For
general HarbourBridge information see <pointer-to-main-README>.

## Example PostgreSQL Usage

Reply:
Done.